pkg/virt-launcher/virtwrap/device/hostdevice/sriov/hostdev_test.go

```diff
@@ -182,6 +235,54 @@ var _ = Describe("SRIOV HostDevice", func() {
 			}
 			Expect(devices, err).To(Equal([]api.HostDevice{expectHostDevice1}))
 		})
+
+		DescribeTable("create two devices with custom boot-order",
+			func(bootOrder1, bootOrder2 uint) {
+				iface1 := newSRIOVInterface(netname1)
+				iface2 := newSRIOVInterface(netname2)
+
+				var expectedBootOrder1 *api.BootOrder
+				var expectedBootOrder2 *api.BootOrder
+
+				if bootOrder1 > 0 {
+					iface1.BootOrder = &bootOrder1
+					expectedBootOrder1 = &api.BootOrder{Order: bootOrder1}
+				}
+
+				if bootOrder2 > 0 {
+					iface2.BootOrder = &bootOrder2
+					expectedBootOrder2 = &api.BootOrder{Order: bootOrder2}
+				}
```

Comment:
This is just magic numbers, I do not understand it at all.
Can you just pass in the `newSRIOVInterface` with the boot-order set?

I do not know how, but this needs to be adjusted so it will not be so cryptic.

Reply:
Changed this function to accept v1.Interface.
Does that works for you?

Reply:
Yea, looks nice now! Thanks.
One nit left.